pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -865,6 +870,19 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListeners(env *model.E
 							Service:          service,
 						}
 
+						if pluginParams.ListenerProtocol == plugin.ListenerProtocolHTTP &&
+							(pilot.HTTP10 || node.Metadata[model.NodeMetadataHTTP10] == "1") {
+							listenerOpts.filterChainOpts = []*filterChainOpts{{
+								httpOpts: &httpListenerOpts{
+									connectionManager: &http_conn.HttpConnectionManager{
+										HttpProtocolOptions: &core.Http1ProtocolOptions{
+											AcceptHttp_10: true,
+										},
+									},
+								},
+							}}
+						}
```

Comment:
this code is unnecessary. The http opts are set in the function below. In fact, this might mess something as you are partially initializing filter chain opts well before its actually initialized by later functions

Reply:
Removed.